style.css

```diff
@@ -80,3 +80,7 @@ section {
   display: flex;
   gap: 1rem;
 }
+
+#socials > div:nth-child(3) {
+  display: flex;
+}
```

Comment:
- [Optional] Dear @Mehr-o-maah, Good job aligning all the items as requested earlier, but this is not the right way to align elements. I would suggest setting a similar width/height to all the elements and applying the `display: flex` property to the container with `justify-content: center` and `align-items: center`. This will make sure your elements will always be at the center.

Reply:
I want to avoid using fixed values, I'll try your suggestion

Reply:
@TadesseDev  can I merge it ? It has not changed the status in the dashboard